addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -142,9 +149,18 @@ export class A11YPanel extends Component<A11YPanelProps, A11YPanelState> {
     });
   };
 
+  onManual = (manual: boolean) => {
+    if (manual) {
+      this.setState({
+        status: 'manual',
+      });
+    } else {
+      this.request();
```

Comment:
Are we sure that the story has been rendered before calling `this.request()`? 🤷‍♂ I'm just asking because `STORY_RENDERED` event was used before

Reply:
It seemed to work for me without problems, but I don't know the internals. Maybe @ndelangen could confirm if events emitted inside `makeDecorator -> wrapper` (see https://github.com/storybookjs/storybook/pull/8883/files/d73733d99e2250d840f35e16044e8de69bdc9b6c#diff-135e5fd1db00e7a987038977b6cb9148R70) happen after rendering? If not, do you know an example I could add to the tests to see when it would fail, so I can fix it?

Reply:
STORY_RENDERED is called after the awaited render function resolved/returned:
https://github.com/storybooks/storybook/blob/38a3dc6ad1d19a98b1be2a7e33b907bbd3252879/lib/core/src/client/preview/start.js#L240-L242

Reply:
So I do believe it's called later. Whether or not this impacts the a11y execution, I don't know.

Reply:
There actually seems to be a problem - not sure it's because of this or something else. If _another_ panel is _initially_ active and you switch to the a11y panel you only see the initial state.

E.g. if you visit https://monorepo-51csjv7sk.now.sh/cra-kitchen-sink/?path=/story/app--full-app for example (taken from a different pull request) and switch to the a11y you see results. While you only see the initial state (at least on _first try_ - if you reload it works) if you run my pull request locally.

I probably need to fix that 😭

Reply:
Good catch @donaldpipowitch, yes we need to fix this, I will try to take a look tonight if I have the time ;)